### PR TITLE
Add default value for `databaseSource` constructor param

### PR DESCRIPTION
**Pull request by llunaCreixent**
Closes https://github.com/CirclesUBI/circles-core/issues/111

### CODE REVIEW

**Comment by llunaCreixent** [comment]
test PASS with:
- api `v1.4.0`
- relayer `v4.1.11`
- subgraph `v1.3.0`

**Review by mikozet — DISMISSED** [review]

**Comment by llunaCreixent** [comment]
test PASS again with:

    api v1.4.0
    relayer v4.1.11
    subgraph v1.3.0

**Review by JacqueGM — APPROVED** [review]

File: test/helpers/core.js [hunk 1]
```diff
@@ -11,6 +11,5 @@ export default function createCore() {
     relayServiceEndpoint: process.env.RELAY_SERVICE_ENDPOINT,
     safeMasterAddress: process.env.SAFE_ADDRESS,
     subgraphName: process.env.SUBGRAPH_NAME,
-    databaseSource: process.env.DATABASE_SOURCE,
```
Comment (JacqueGM) [inline test/helpers/core.js:14]: without this vr in the tests?

Comment (llunaCreixent) [reply test/helpers/core.js:14]: exactly, because I added a default value, and therefore is not mandatory

Comment (llunaCreixent) [reply test/helpers/core.js:14]: test pass even in the GitHub Actions

**Pull request merged**